scenes/chainer-caption/data/MSCOCO/mscoco_caption_train2014_processed_dic.json

```diff
@@ -0,0 +1,8852 @@
+    "exit": 6844, 
+    "exiting": 2889, 
+    "exits": 7543, 
+    "exotic": 3517, 
+    "expanse": 2947, 
+    "expansive": 7461, 
+    "expensive": 1455, 
+    "experience": 304, 
+    "experiment": 3193, 
+    "expert": 1448, 
+    "expertly": 121, 
+    "expired": 5390, 
+    "explaining": 3753, 
+    "exploring": 8486, 
+    "expo": 3947, 
+    "exposed": 902, 
+    "exposure": 562, 
+    "express": 1438, 
+    "expressing": 7936, 
+    "expression": 2380, 
+    "expressions": 1356, 
+    "extend": 79, 
+    "extended": 1321, 
+    "extending": 7231, 
+    "extends": 3281, 
+    "extension": 4373, 
+    "extensive": 7926, 
+    "exterior": 6117, 
+    "external": 7373, 
+    "extinguisher": 4198, 
+    "extra": 713, 
+    "extraordinary": 1406, 
+    "extravagant": 390, 
+    "extreme": 8128, 
+    "extremely": 5820, 
+    "eye": 5759, 
+    "eyeballs": 6051, 
+    "eyed": 5651, 
+    "eyeglasses": 8076, 
+    "eyeing": 5043, 
+    "eyes": 5650, 
+    "f": 5919, 
+    "fabric": 124, 
+    "fabrics": 8361, 
+    "fabulous": 6038, 
+    "facade": 5861, 
+    "face": 4861, 
+    "faced": 8367, 
+    "faces": 8371, 
+    "facet": 8369, 
+    "facial": 2105, 
+    "facility": 4509, 
+    "facing": 5709, 
+    "factory": 8474, 
+    "faded": 426, 
+    "fading": 8030, 
+    "failing": 7226, 
+    "faint": 6321, 
+    "fair": 2232, 
+    "fairly": 2249, 
+    "fairy": 861, 
+    "fake": 4553, 
+    "falcon": 6667, 
+    "fall": 3626, 
+    "fallen": 1924, 
+    "falling": 6714, 
+    "falls": 4098, 
+    "false": 493, 
+    "families": 2824, 
+    "family": 5220, 
+    "famous": 3109, 
+    "fan": 8593, 
+    "fancily": 2689, 
+    "fancy": 8634, 
+    "fans": 5837, 
+    "far": 8588, 
+    "farm": 971, 
+    "farmer": 1780, 
+    "farmers": 6237, 
+    "farmhouse": 7688, 
+    "farming": 4990, 
+    "farmland": 5379, 
+    "farther": 5036, 
+    "fascinated": 7362, 
+    "fascinating": 5540, 
+    "fashion": 3949, 
+    "fashionable": 213, 
+    "fashioned": 3442, 
+    "fast": 4270, 
+    "fastened": 4532, 
+    "faster": 1744, 
+    "fat": 8590, 
+    "father": 1230, 
+    "fatigues": 6706, 
+    "faucet": 1409, 
+    "faucets": 2040, 
+    "faux": 6497, 
+    "favorite": 6847, 
+    "fe": 3063, 
+    "feast": 6481, 
+    "feasting": 1434, 
+    "feather": 516, 
+    "feathered": 4075, 
+    "feathers": 1620, 
+    "feathery": 1619, 
+    "feature": 4415, 
+    "featured": 5503, 
+    "features": 5500, 
+    "featuring": 1822, 
+    "fed": 4290, 
+    "fedex": 2728, 
+    "fedora": 5300, 
+    "feed": 6355, 
+    "feeder": 4707, 
+    "feeders": 5686, 
+    "feeding": 5463, 
+    "feeds": 605, 
+    "feel": 6357, 
+    "feeling": 56, 
+    "feet": 2283, 
+    "feild": 5318, 
+    "fell": 3451, 
+    "fellow": 7956, 
+    "felt": 3449, 
+    "female": 1606, 
+    "females": 4408, 
+    "feminine": 1218, 
+    "fence": 7425, 
+    "fenced": 8718, 
+    "fenced-in": 8513, 
+    "fences": 8720, 
+    "fencing": 7675, 
+    "fende
```

Comment:
Provide a Arabic version of this file, you can use [this code](https://github.com/cane-see-project/arabic_translator) for translation.